### DISCUSSION TITLE
My Youtube channel for serenity tutorials and coding

### DISCUSSION THREAD

**Discussion by imhemayatsangin** [serenity-is/Serenity#6353] [Tricks and Samples]
Good morning folks and serenity lovers!
Hope you are doing well. I have created a serenity tutorial playlist and I will be adding serenity tricks, skills and Samples and specially  whatever I practice and I hope it will help you guys. Please subscribe my channel for more videos and if you like thumbs it up.

[Serenity Framework tricks and snippets ](https://www.youtube.com/playlist?list=PLVjb55Wh0HEAuJxTEmBU9ZiYLCaPp4O06)

Regards,
Hemayat Sangin

**Comment by mvmshakkir** [comment]
Not found
